apps/src/gamelab/AnimationPicker/AnimationPickerBody.jsx

```diff
@@ -3,14 +3,24 @@ import React from 'react';
 import Radium from 'radium';
 import Immutable from 'immutable';
 import color from '../../color';
+import {AnimationCategories} from '../constants';
 import gamelabMsg from '@cdo/gamelab/locale';
 import animationLibrary from '../animationLibrary.json';
 import ScrollableList from '../AnimationTab/ScrollableList.jsx';
 import styles from './styles';
 import AnimationPickerListItem from './AnimationPickerListItem.jsx';
 import AnimationPickerSearchBar from './AnimationPickerSearchBar.jsx';
 
-const MAX_SEARCH_RESULTS = 18;
+const MAX_SEARCH_RESULTS = 40;
+const allAnimationsStyle = {
+  color: color.purple,
+  fontFamily: "'Gotham 7r', sans-serif, sans-serif",
+  cursor: "pointer"
+};
+const breadCrumbsStyle = {
+  margin: "8px 0",
+  fontSize: 14
+};
```

Comment:
Not sure if you guys take this approach in GL, but in much of our code, we've been putting all of our styles in a single obj, i.e.
```
const styles = {
  allAnimations: {
    color: color.purple,
    ...
  },
  breadCrumbs: {
    margin: "8x 0"
    fontSize: 14
  }
}